content/en/docs/distributions/openshift/install-kubeflow.md

```diff
@@ -39,7 +39,10 @@ Minimal:
 
 Use the following steps to install Kubeflow 1.0 on OpenShift 4.x.
```

Comment:
not related to your change, but notice it says Kubeflow `1.0` here.

Reply:
yes fixed it